hibernate-core/src/test/resources/META-INF/services/org.hibernate.boot.model.TypeContributor

```diff
@@ -0,0 +1 @@
+org.hibernate.orm.test.type.contributor.usertype.ServiceLoadedCustomUserTypeTypeContributor
```

Comment:
Please use `@ServiceRegistry(javaServices = @JavaService(role = TypeContributor.class, impls = ServiceLoadedCustomUserTypeTypeContributor.class))` on the particular test instead.